caps/management/commands/preprocess.py

```diff
@@ -145,26 +145,51 @@ def replace_headers():
         settings.RAW_CSV,
         [
             "council",
-            "search_link",
-            "unfound",
-            "credit",
             "url",
             "date_retrieved",
-            "time_period",
             "type",
-            "scope",
-            "status",
-            "homepage_mention",
-            "dedicated_page",
-            "well_presented",
-            "baseline_analysis",
-            "notes",
-            "plan_due",
             "title",
-            "title_checked",
```

Comment:
This row was used to see if the title should be updated. without it all the document title's get set to blank.

The `get_plan_defaults_from_row` function in `import_plans` will need to be updated to account for this.

Reply:
The test is `if char_from_text(row["title_checked"]).lower() == "y":` so Y or y would work

Reply:
Have removed what was checking this later on.